### DISCUSSION TITLE
After sign in we are not receiving users data back from supabase

### DISCUSSION THREAD

**Discussion by Kitellc** [supabase/supabase#7645] [Questions]
Hey,

We just finished our Auth Integration for different social media platforms Linkedin, Twitch, Discord using Supabase. 
So, firstly I really wanna thank you for creating supabase.

I am facing some problems too i.e. Once the users enter the login info, it's getting stored in the Supabase dashboard but we are unable to show that info onto our app. 

Can you please help me out with this?

**Comment by GaryAustin1** [comment]
Assuming you are trying to read the auth.user table from the API, you can't do that.
You can write an RPC function assigned "security definer" type to read the auth.users table (make sure you handle any security you need in the function).
Or you can use the recommended way of creating a public profile/user table with copies of what you need.  This is done with an update/insert trigger on auth.users table and inserting into your own table.  https://supabase.com/docs/guides/auth/managing-user-data
